.gitignore

```diff
@@ -136,3 +136,6 @@ generated*
 **/mlruns/
 **/runs/
 **/.yalc/
+
+#pylint
+.pylintrc
```

Comment:
I have no idea what this file does ... @frascuchon do we still need this? Can we safely delete it?
Anyways, i don't think we should mention it in `.gitignore`:
```suggestion
```

Reply:
This is the configuration file that the Python linter pylint produces sometimes. It is a very common linter, so I thought it could be useful to add it to the .gitignore. Anyway, we can erase this change.

Reply:
ok, i see, let's wait for @frascuchon input just in case ...

Reply:
Since we set linter configuration in `setup.cfg` I think we could drop references to `.pylintrc` file. But I have no idea what better linter config way.

Reply:
Then, I think we can exclude this change and maybe address it later